crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -811,6 +811,7 @@ en:
     not_found: 'Not found'
     chef_server_down: 'Chef server is not available'
     param_missing: 'A required parameter is missing'
+    before_install: 'Some of the API is not available before installation'
```

Comment:
"API is not available before installation" is certainly better, but not really true, as it is only part of it that is not available.
It shall not mislead the user to think he can not use the `crowbarctl` at all.

would this be ok for you?